app/src/main/java/com/nervousfish/nervousfish/activities/SettingsActivity.java

```diff
@@ -41,7 +45,8 @@
 public final class SettingsActivity extends AAppCompatPreferenceActivity {
 
     private static final Logger LOGGER = LoggerFactory.getLogger("SettingsActivity");
-    private static boolean firstLoad = true;
+    private static boolean firstLoadName = true;
+    private static boolean firstLoadIban = true;
```

Comment:
For loading the first time, we should get the data from the database. Otherwise we could get them from the filled in preference